codemirror-ls/src/extensions/linting.ts

```diff
@@ -199,7 +204,9 @@ export const getLintingExtensions: LSExtensionGetter<DiagnosticArgs> = ({
             return;
           }
 
-          view.dispatch(setDiagnostics(view.state, resolvedDiagnostics));
+          return await this.processQueue.add(() => {
+            view.dispatch(setDiagnostics(view.state, resolvedDiagnostics));
```

Comment:
I'm not sure that this would work in theory - it's a promise queue and it's serializing, but the `view.dispatch` method is synchronous so wouldn't we end up with the same ordering and timing of view.dispatch calls as we did before? p-queue would make more sense if this method returned a promise and we wanted one method to start after the last method's promise resolved, but in this case the method will return immediately.